tests/calc/test_indices.py

```diff
@@ -130,16 +131,15 @@ def test_weighted_continuous_average():
     assert_almost_equal(v, 6.900543760612305 * units('m/s'), 7)
 
 
-@pytest.mark.xfail(reason='hgrecco/pint#1593')
+@pytest.mark.xfail(condition=module_version_before('pint', '0.21'), reason='hgrecco/pint#1593')
 def test_weighted_continuous_average_temperature():
     """Test pressure-weighted mean temperature function with vertical interpolation."""
     data = get_upper_air_data(datetime(2016, 5, 22, 0), 'DDC')
     t, = weighted_continuous_average(data['pressure'],
                                      data['temperature'],
                                      height=data['height'],
                                      depth=6000 * units('meter'))
-    # Commenting out since it won't run until the above can run without error
-    # assert_almost_equal(t, 279.3275828240889 * units('kelvin'), 7)
+    assert_almost_equal(t, 279.07450928270185 * units('kelvin'), 7)
```

Comment:
I'm unsettled that when I had made this change locally, it wasn't failing...

Reply:
Now failing for me with `main`, so I guess something recently changed this. Trying to figure out if I want to know what it was.

Reply:
Nevermind, back to at least 1.4.0 these values lead to a failure (uncommenting and removing the xfail), so I guess things didn't change at all "recently". Happy to just see this updated.